src/lib/readmeAPIFetch.ts

```diff
@@ -367,8 +367,8 @@ export async function handleAPIv1Res(
  *
  * If we receive non-JSON responses, we consider them errors and throw them.
  */
-export async function handleAPIv2Res(
-  this: CommandClass['prototype'] | Hook.Context,
+export async function handleAPIv2Res<T extends Hook.Context = Hook.Context>(
```

Comment:
`Hook.Context` also works for `CommandClass['prototype']`?

Reply:
it's actually quite small and generic and contains just `config` and logging properties like a command does:

https://github.com/oclif/core/blob/d4728f764350907d1187f837e8190f5b96285536/src/interfaces/hooks.ts#L129

Reply:
i'll add a little comment so that's clearer